CMakeLists.txt

```diff
@@ -287,6 +288,8 @@ set_target_properties(rocksdb${ARTIFACT_SUFFIX} PROPERTIES COMPILE_FLAGS "-DROCK
 add_dependencies(rocksdb${ARTIFACT_SUFFIX} GenerateBuildVersion)
 target_link_libraries(rocksdb${ARTIFACT_SUFFIX} ${LIBS})
 
+include(${CMAKE_CURRENT_SOURCE_DIR}/java/jni.inc)
```

Comment:
1. Ok I can do that
2. I am reusing setup, variables, targets from main cmake. That us why I used include. I did not want to loose changes that anyone will do in main cmake. JNI is just little bit different version of rocksdb.dll. I do not think it is different project, but also not third party. Include is easiest way how to achieve that.